### ISSUE TITLE
jwm -f FILENAME -restart

### ISSUE THREAD

**Issue by Israel-D** [joewing/jwm#444]
I expect that running `jwm -f "$FILENAME" -restart` (with a valid file) should restart jwm using the file specified.
This does not occur, is there a way to do this type of thing?

**Comment by Israel-D** [comment]
@joewing It just seemed like it could be an interesting way to make really interactive programs.  It could be used as some sort of introductory screen where the user could make choices, and then jwm could load the default jwmrc file.
It could even be used as some sort of OEM feature where the first user is allowed to add a new user, configure things, etc..

My main use was to provide a graphical menu screen to choose between options of things needed, with something like a tray in the middle of the screen and traybuttons that could choose what to start, or even if the user wanted to exit to the shell.

**Comment by technosaurus** [comment]
Does -reload -restart work?  ...posting from my phone.

I think this is probably the intended behavior, we just need to check if it is a different file or if the modified time has changed and if so, reload it.  Need to either save the modified time or just reload it unconditionally if a file is specified during a restart.   The existing behavior probably saves some noticeable delay for the common case though.

**Comment by Israel-D** [comment]
@technosaurus No, those options did not work AFAIK this feature does not exist yet.  It will only load a new file if you exit, and restart.
I think this is a feature you could come up with some creative uses for 😄

**Comment by technosaurus** [comment]
I already used the reload feature to have a drive menu and package manager, but dynamic menus made those obsolete.  I don't think this will be hard to change... just allow reload and restart... Then we just need send event support to allow dropping files to menu items
